clang/lib/Tooling/DependencyScanning/ScanAndUpdateArgs.cpp

```diff
@@ -13,12 +13,24 @@
 #include "clang/Tooling/DependencyScanning/DependencyScanningTool.h"
 #include "llvm/CAS/CachingOnDiskFileSystem.h"
 #include "llvm/CAS/ObjectStore.h"
+#include "llvm/Support/Path.h"
 #include "llvm/Support/PrefixMapper.h"
+#include "llvm/Support/VirtualFileSystem.h"
```

Comment:
Is this header used by something? I only see `llvm::sys::path` being added in this PR